src/condor_job_router/JobRouter.cpp

```diff
@@ -1037,6 +1037,9 @@ JobRouter::EvalSrcJobPeriodicExpr(RoutedJob* job)
 			// the job as if it is a new job from the schedd.
 			ret_val = true;
 			break;
+		case VACATE_FROM_RUNNING:
+			SetJobIdle(job);
```

Comment:
SetJobIdle() just sets the JobStatus of the source job to Idle. I think the Job Router should treat VACATE_FROM_RUNNING the same as STAYS_IN_QUEUE (i.e. do nothing). The job policy expressions are forwarded to the routed job, where PeriodicVacate can have its intended effect. Evaluating the policy expressions in both the unrouted and routed jobs is highly suspect to me, but that's an issue for another ticket.